api/build.gradle.kts

```diff
@@ -8,6 +8,7 @@ dependencies {
     compileOnly("io.papermc.paper:paper-api:${findProperty("minecraftVersion")}-R0.1-SNAPSHOT")
 
     compileOnly("de.oliver:FancyLib:${findProperty("fancyLibVersion")}")
+    compileOnly("org.geysermc.geyser:api:${findProperty("geyserVersion")}")
```

Comment:
I am unsure if this will support all Geyser distributions - as far as I am aware this api would only work for Geyser Spigot/Paper, I would suggest depending on Floodgate instead.